nx.json

```diff
@@ -71,7 +71,7 @@
     "@fluentui/jest-serializer-make-styles": { "implicitDependencies": [] },
     "@fluentui/jest-serializer-merge-styles": { "implicitDependencies": [] },
     "@fluentui/keyboard-key": { "implicitDependencies": [] },
-    "@fluentui/make-styles": { "implicitDependencies": [] },
+    "@fluentui/make-styles": { "tags": ["vNext", "platform:web"], "implicitDependencies": [] },
```

Comment:
to double check the context boundary, is this package solely for `web` platform or it can be used also within `node` ?

Reply:
It's a web package, some functionality is related to SSR, but I am not considering this usage as within Node.

Reply:
kk, for better context 

- platform:web - package used on web
- platform:node - package used with node
- platform:any - package that can be used in both

We'll enable lint rule that will respect those contexts and staticly defined rules in the future to prevent for example importing `node` package into `web` package